app/src/main/java/org/kiwix/kiwixmobile/zim_manager/Fat32Checker.kt

```diff
@@ -42,7 +42,7 @@ class Fat32Checker constructor(
 
   init {
     Flowable.combineLatest(
-      sharedPreferenceUtil.prefStorages
+      sharedPreferenceUtil.getPrefStorages()
```

Comment:
@macgills Doing this would cause some problem(although the app run fine)? This change was made as the old one wasn't working

Reply:
What problems is it causing?
The issue here was kotlin java interop, in kotlin if a java class has methods `getProperty` & `setProperty(someValue)` then you can use `property access syntax` to get and set that property eg
java
```
Object aValue = anObject.getProperty()
anObject.setProperty(newValue)
```
kotlin
```
val aValue = anObject.property
anObject.property = newValue
```

Reply:
No, it didn't cause any problem but I was thinking if this file could remain unchanged

Reply:
to leave it unchanged what is common in kotlin is to use a private value prefixed with an `_` internally and then have a public value without the `_` to expose the value as a different type or modified value

eg
```
private val _prefStorages = PublishProcessor.create(...)
val prefStorages = _prefStorages.startWith(...)
```

Reply:
Oh so you mean backing, right?
Okay fine!!

Reply:
well I tried to avoid the term `backing` in my description to keep it simple because there are 2 possible interpretations of that in kotlin, a `backing field` demonstrated above or a `backing property`, they can be read about [here](https://kotlinlang.org/docs/reference/properties.html#backing-fields)

Reply:
@s-ayush2903 this wasn't done, the end result is this file should be unchanged